tasklist/app/views/people/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<h1>People List</h1>
+<table>
+	<th>Name</th><th># Complete Tasks</th><th colspan="2">Update!</th>
+	<% @people.each do |person| %> 
+		<tr>
+			<td>
+				<%= link_to "#{person.name}", "/people/#{person.id}" %>
+			</td>
+			<td>
+				<% complete_count = 0 %> 
+				<% person.tasks.each do |task| %>
```

Comment:
Is there a simpler way to get this count by using `filter` or `reject` and `count`?